files.h

```diff
@@ -11,6 +11,6 @@ namespace files {
 	void init();
 }
 
-static bool endsWith(const std::string &str, const std::string &suffix) {
+[[maybe_unused]] static bool endsWith(const std::string &str, const std::string &suffix) {
```

Comment:
I did notice this was unused too when I was tinkering with wibo, it built fine when I removed it, idk what others think

Reply:
It's used in one source file, but in the other source files, it's marked as unused, since it's in a header. There's probably a better way to fix this.